docs/cli/migrate.md

```diff
@@ -0,0 +1,39 @@
+---
+id: migrate
+title: "Migrating from Botkube Open Source"
+sidebar_position: 2
+---
+
+## Migrating from Botkube Open Source
```

Comment:
Maybe it should be under some "guides" or even "cloud" category? or I should change the title?

Reply:
🤔 let's postpone this decision for now, I think it is OK as it is right now unless we have more docs related to Cloud 👍